app/src/main/java/de/westnordost/streetcomplete/view/controller/AutoCorrectAbbreviationsViewController.kt

```diff
@@ -47,7 +47,7 @@ class AutoCorrectAbbreviationsViewController(private val editText: EditText) {
             fixedReplace(s, wordStart, wordStart + lastWordBeforeCursor.length, replacement)
         } else if (lastWordBeforeCursor.length > 3) {
             val locale = abbrs.locale
-            val capital = lastWordBeforeCursor.substring(0, 1).uppercase(locale)
+            val capital = lastWordBeforeCursor.get(0).titlecase(locale)
             s.replace(wordStart, wordStart + 1, capital)
```

Comment:
Uh wait how can this even compile? `String.get(0)` returns a `Char`, which is fine because `titlecase` is a method of `Char`. But `s` is a `Editable` and the `replace` method expects a `CharSequence` (implemented by `String`), not a `Char`:

https://developer.android.com/reference/android/text/Editable#replace(int,%20int,%20java.lang.CharSequence)

Reply:
Oh, nevermind. `titlecase` returns a `String`. That's fine then. Thank you!

Reply:
`titlecase` returns a `String`, since in some languages title-casing a single letter may result in multiple letters.